src/register/register.ts

```diff
@@ -52,6 +52,27 @@ export class Register {
       clipboard.copy(content);
     }
 
+    // Upper-case register name denotes append.
+    if (/^[A-Z]$/.test(register)) {
```

Comment:
Can you move this into it's own separate static function (`isNamedRegister`), similar to `isValidRegister`?